src/client/components/BackgroundImage.tsx

```diff
@@ -0,0 +1,76 @@
+import React, { useState } from 'react'
+import styled from '@emotion/styled'
+import { css } from '@emotion/core'
+import { colorsV3 } from '@hedviginsurance/brand'
+import { LARGE_SCREEN_MEDIA_QUERY } from 'utils/mediaQueries'
```

Comment:
I missed that we had defined these - nice!

Reply:
Yep, I added them a while back but it's only me using them I think lol